.github/workflows/sync-teams.yml

```
@@ -0,0 +1,41 @@
+name: '[Support] Synchronize team members in the .env file'
+on:
+  schedule:
+    # Daily
+    - cron: '0 1 * * *'
+permissions:
+  repository-projects: write
+
+jobs:
+  sync-support-teams:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repo checkout
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.BITNAMI_BOT_TOKEN }}
+          fetch-depth: 1
+      - name: Load .env file
+        uses: xom9ikk/dotenv@v1.0.2
+        with:
+          path: .github/workflows/
+      - name: Updating members of the Bitnami team
+        run: |
+          TEAM_MEMBERS=$(curl --request GET \
+          --url https://api.github.com/repos/orgs/bitnami/teams/developers/members \
+          --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
+          --header 'content-type: application/json' \
+          | jq 'sort_by(.login)|map(.login)|join(",")') \
```

I think last `\` is not needed
```suggestion
          | jq 'sort_by(.login)|map(.login)|join(",")')
```